orttraining/orttraining/python/training/onnxblock/checkpoint_utils.py

```diff
@@ -2,6 +2,9 @@
 # Licensed under the MIT License.
 # checkpoint_utils.py
 
+from onnx import TensorProto
+
+from onnxruntime.capi._pybind_state import load_checkpoint as _internal_load_checkpoint
```

Comment:
Combine the imports:
```py
from onnxruntime.capi._pybind_state import load_checkpoint as _internal_load_checkpoint, save_checkpoint as _internal_save_checkpoint
```

Reply:
just curious: why "load_checkpoint as _internal_load_checkpoint" why not just import load_checkpoint?